distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -152,6 +152,8 @@ grant codeBase "file:/jetty-server/catalog-core-directorymonitor/catalog-securit
 }
 
 grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
```

Comment:
❓ Is this another necessary permission that came up after the keystore permission was added?

Reply:
Yes, it is. The line from `getSystemSubject` which calls out to the AbstractStsRealm, which needs access to the properties files in `etc/ws-security`: https://github.com/codice/ddf/blob/f3c7964ebadcf31cc96a7b37ada4b08a2ffe7afb/platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java#L224